hw/top_earlgrey/data/top_earlgrey.hjson

```diff
@@ -566,8 +566,8 @@
       reset_connections: {rst_ni: "sys"},
       base_addr: "0x41110000",
     },
-    { name: "kmac"
-      type: "kmac"
+    { name: "kmac",
+      type: "kmac",
```

Comment:
ah sorry, this is mostly because of the editor i'm using. It doesn't indent the next line nicely unless the previous line has a comma. Let me remove this.

Reply:
just curious, what editor do people use? i'm a vanilla emacs user..but wondering if i should be updating...

Reply:
Emacs here too. For hjson, I've given up on clever modes, so tend to put it in C mode for the paren matching and avoid hitting tab... For languages that anyone else uses, Emacs is brilliant.